internal/commands/push/inputs.go

```diff
@@ -9,15 +9,6 @@ import (
 	"github.com/10gen/realm-cli/internal/utils/flags"
 )
 
-const (
-	flagLocalPath           = "local"
```

Comment:
again i would keep these here.  basically, we're standardizing the names _just_ for `push`